scripts/install_azcopy.sh

```diff
@@ -0,0 +1,7 @@
+echo "Downloadng azcopy ...."
+wget https://aka.ms/downloadazcopy-v10-linux 
+tar -xvf downloadazcopy-v10-linux
+sudo rm /usr/bin/azcopy
```

Comment:
>/usr/bin/ [](http://example.com/codeflow?start=8&length=9)

Why does azcopy need to be in /usr/bin? 

Can we not execute it from the downloaded folder itself?

Reply:
Also, please prefer `/opt/`

Reply:
sure